examples/03-Circuit/EMIT_Example.py

```diff
@@ -37,15 +36,18 @@
 # Create 3 radios and connect an antenna to each.
 rad1 = aedtapp.modeler.components.create_component("UE - Handheld")
 ant1 = aedtapp.modeler.components.create_component("Antenna")
-ant1.move_and_connect_to(rad1)
+if rad1 and ant1:
```

Comment:
Under what circumstances will this be false? In that situation does it make sense to proceed with the example?

I think it would only happen when there is an error with create_component. If there are errors, I think it may be better to fail sooner rather than later.

Reply:
Hi @myoung301 , it is false when running in non-graphical. If you notice the latest version of the documentation has not been updated because the doc build runs in non-graphical. With this trick the documentation should be generated and the example should run on customer machine